.circleci/config.yml

```diff
@@ -14,11 +14,15 @@ executors:
       GATSBY_CPU_COUNT: 2
 
 aliases:
+  e2e-executor-env: &e2e-executor-env
+    GATSBY_CPU_COUNT: 2
+    VERBOSE: 1
```

Comment:
I am adding that `VERBOSE` flag here for e2e tests (it is helpful sometimes to have that extra logs around when debugging failures, sometimes there are some clues)